configuration/reports/reportdescriptors/dataexports/ncd.yml

```diff
@@ -0,0 +1,30 @@
+key: "ncdexport"
+uuid: "13629220-6b9e-11e5-a837-0800200c9a66"
+name: "mirebalaisreports.ncddataexport.name"
```

Comment:
not 100% sure, but I don't think there's any reason to continue using "mirebalaisreports" in the name/description of exports - especially ones that won't be used in Haiti.